apps/cowswap-frontend/src/modules/tokensList/hooks/useOpenTokenSelectWidget.ts

```diff
@@ -18,10 +20,13 @@ export function useOpenTokenSelectWidget(): (
 ) => void {
   const updateSelectTokenWidget = useUpdateSelectTokenWidgetState()
   const closeTokenSelectWidget = useCloseTokenSelectWidget()
+  const isSmartContractWallet = useIsSmartContractWallet()
+  const isBridgingEnabled = useIsBridgingEnabled(isSmartContractWallet)
```

Comment:
Can `useIsSmartContractWallet()` be defined inside of `useIsBridgingEnabled`?

Reply:
Circular dependency, unfortunately.
Could probably be refactored, but it was breaking the wallets and tokens libs.